incubator/elasticsearch-exporter/README.md

```diff
@@ -0,0 +1,80 @@
+# Elasticsearch Exporter
+
+Prometheus exporter for various metrics about ElasticSearch, written in Go.
+
+Learn more: https://github.com/justwatchcom/elasticsearch_exporter
+
+## TL;DR;
+
+```bash
+$ helm install .
+```
+
+## Introduction
+
+This chart creates a Elasticsearch-Exporter deployment on a [Kubernetes](http://kubernetes.io) 
+cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.5+ with Beta APIs enabled
```

Comment:
1.8+

We officially support the current and previous minor version of Kubernetes.